mdbenchmark/analyze.py

```diff
@@ -132,15 +133,27 @@ def plot_analysis(df, ncores):
     help='Number of cores per node. If not given it will be parsed from the '
     'benchmarks log file.',
     show_default=True)
-def analyze(directory, plot, ncores):
+@click.option(
+    '-o', '--output-name', default=None, help="Name of the output .csv file.", type=str)
+def analyze(directory, plot, ncores, output_name):
     """Analyze finished benchmarks."""
     bundle = mds.discover(directory)
+
     df = pd.DataFrame(columns=[
-        'gromacs', 'nodes', 'ns/day', 'run time [min]', 'gpu', 'host', 'ncores'
+        'module', 'nodes', 'ns/day', 'run time [min]', 'gpu', 'host', 'ncores'
```

Comment:
If we want to stay backward compatible we should keep the `gromacs` column. Any other module then gromacs would be stored with a '?' in this columns. The `module` column should still be added. This would ensure that plotting scripts in the wild based on the old files continue to work.